frontend/app/mixins/graphable.js

```diff
@@ -0,0 +1,94 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
```

Comment:
do you mean to create some component only to extend it with `g-trackable` and `g-weather`?